test/test_transforms_tensor.py

```diff
@@ -464,6 +464,7 @@ def test_resized_crop_save(self, tmpdir):
 
 
 def _test_random_affine_helper(device, **kwargs):
+    torch.manual_seed(12)
```

Comment:
I had to add this line and fix shear value due to some random jit vs non-jit results mismatch in one pixel